cfme/cloud/tenant.py

```diff
@@ -92,7 +93,8 @@ def in_tenants(self):
         return (
             self.logged_in_as_current_user and
             self.navigation.currently_selected == ['Compute', 'Clouds', 'Tenants'] and
-            # TODO: Needs to be converted once there's a Widgetastic alternative
+            # TODO: Needs to be converted once there's a Widgetastic
```

Comment:
Again don't split comment lines unless they're over 100 chars and lint is complaining.